src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -541,10 +572,63 @@ public Visitable visit(MethodCallExpr method, Void p) {
     return super.visit(method, p);
   }
 
+  @Override
+  public Visitable visit(ClassOrInterfaceType typeExpr, Void p) {
+    /**
+     * Workaround for a JavaParser bug: When a type is referenced as a class path, like
```

Comment:
"as a class path" is imprecise here. I think you mean "when a type is referenced by its fully-qualified name"